tests/id-test.js

```diff
@@ -39,11 +40,12 @@ test('Recreate an B58 String', function (t) {
 
 test('Recreate from a Public Key', function (t) {
   var id = PeerId.createFromPubKey(testId.pubKey)
-  t.ok(id)
+  t.ok(id.pubKey == testId.pubKey)
```

Comment:
this only works because a `==` passed. Buffers have to be compared with `.equals`, see https://nodejs.org/api/buffer.html#buffer_buf_equals_otherbuffer

As a note, tape has a t.equal(a, b) for normal types. For objects and Buffers comparisons, use t.deepEqual